convert2rhel/pkghandler.py

```diff
@@ -661,7 +661,7 @@ def remove_non_rhel_kernels():
 
 def fix_default_kernel():
     """
-    Systems converted from Oracle Linux or CentOS Plus may have leftover kernel-uek or kernel-plus in
+    Systems converted from Oracle Linux or CentOS Linux Plus may have leftover kernel-uek or kernel-plus in
```

Comment:
```suggestion
    Systems converted from Oracle Linux or CentOS Linux may have leftover kernel-uek or kernel-plus in
```
As it was before, it suggested the there's such a thing as an OS named CentOS Linux Plus. But in reality there's just CentOS Linux on which you can enable the Plus repository and install the kernel-plus package from there.